### DISCUSSION TITLE
Is there any supported TRV which supports setting externally measured temperature as current?

### DISCUSSION THREAD

**Discussion by antst** [Koenkk/zigbee2mqtt#10152] [Devices]
I am looking TRV which can use measured externally and supplied temperature instead measured on device.
Of course, there is a trick with automation and constantly changing calibration.  And this is way better than all this emulations which hits 0.5C steps anyway (this, affecting bad way attempts to reach equilibrium).

My old crappy eq3 TRVs which used to cost next to nothing, can do that and I am surprised that most of Zigbee2MQTT TRVs can not.

**Comment by mipoat** [comment]
Danfoss Ally and Popp can do that. Have a look for "External_measured_room_sensor (numeric)" here:
https://www.zigbee2mqtt.io/devices/014G2461.html
https://www.zigbee2mqtt.io/devices/701721.html
